src/main/java/frc/robot/RobotContainer.java

```diff
@@ -5,130 +5,108 @@
-            .withVelocityY(-joystick.getLeftX() * MaxSpeed) // Drive left with negative X (left)
-            .withRotationalRate(-joystick.getRightX() * MaxAngularRate) // Drive counterclockwise with negative X (left)
-        ));
-
-    joystick.a().whileTrue(drivetrain.applyRequest(() -> brake));
-    joystick.b().whileTrue(drivetrain
-        .applyRequest(() -> point.withModuleDirection(new Rotation2d(-joystick.getLeftY(), -joystick.getLeftX()))));
-
-    // reset the field-centric heading on left bumper press
-    joystick.leftBumper().onTrue(drivetrain.runOnce(() -> drivetrain.seedFieldRelative()));
-
-    if (Utils.isSimulation()) {
-      drivetrain.seedFieldRelative(new Pose2d(new Translation2d(), Rotation2d.fromDegrees(90)));
+    
+    // Gamepad and generic initialization
+    
+    private final CommandXboxController gamepad_ =
+        new CommandXboxController(OperatorConstants.kDriverControllerPort);
+    
+    private double maxSpeed_ = TunerConstants.kSpeedAt12VoltsMps; // kSpeedAt12VoltsMps desired top speed
+    private double maxAngularRate_ = 1.5 * Math.PI; // 3/4 of a rotation per second max angular velocity
+    
+    private final SwerveRequest.FieldCentric drive_ = new SwerveRequest.FieldCentric()
+    .withDeadband(maxSpeed_ * 0.1).withRotationalDeadband(maxAngularRate_ * 0.1) // Add a 10% deadband
+    .withDriveRequestType(DriveRequestType.OpenLoopVoltage); // I want field-centric driving in open loop
+    
+    private final SwerveRequest.SwerveDriveBrake brake_ = new SwerveRequest.SwerveDriveBrake();
+    private final SwerveRequest.PointWheelsAt point_ = new SwerveRequest.PointWheelsAt();   
+    private final Telemetry swerveTelemetry_ = new Telemetry(maxSpeed_);
+    
+    // Subsystems
+    
+    private final CommandSwerveDrivetrain drivetrain_ = TunerConstants.DriveTrain; 
+
+    private final IntakeShooterSubsystem intake_shooter_ =
+        new IntakeShooterSubsystem(new IntakeShooterIOHardware());  
+
+    private final OISubsystem oiPanel_ = new OISubsystem(2);   
+    
+    /** The container for the robot. Contains subsystems, OI devices, and commands. */
+    public RobotContainer() {
+        // Configure the trigger bindings
+        configureBindings();
+        setupDrivetrain();
+    }
+    
+    /**
+    * Use this method to define your trigger->command mappings. Triggers can be created via the
+    * {@link Trigger#Trigger(java.util.function.BooleanSupplier)} constructor with an arbitrary
+    * predicate, or via the named factories in {@link
+    * edu.wpi.first.wpilibj2.command.button.CommandGenericHID}'s subclasses for {@link
+    * CommandXboxController Xbox}/{@link edu.wpi.first.wpilibj2.command.button.CommandPS4Controller
+    * PS4} controllers or {@link edu.wpi.first.wpilibj2.command.button.CommandJoystick Flight
+    * joysticks}.
+    */
+    private void configureBindings() {
```

Comment:
How come you deleted the OI indicator bindings?

Reply:
Because they were only for testing the OI class which I had already tested.